data/common.yaml

```diff
@@ -0,0 +1,14 @@
+---
+x2go::server::agent_options:
```

Comment:
There should be a `lookup_options` setting for each hash (unless you didn't want to use merging)